src/__tests__/organizations.int.test.ts

```diff
@@ -370,9 +368,9 @@ describe('/organizations', () => {
 				value: 'invalid-email-address.com',
 				isValid: false,
 			});
-			await agent
-				.set(authHeader)
+			await request(app)
 				.get(`/organizations/${organizationId}`)
+				.set(authHeader)
```

Comment:
This little re-ordering of the `set(authHeader)` call makes all the difference. When setting it on the `agent` (as in the previous commit), that makes the headers persist on that agent. When setting it on this (new/fresh) request returned by the call to `.get(...)`, then those headers are *not* re-used on the agent. It's a subtle but important difference. Despite the mistake, I think that to avoid such a mistake in future, we should not introduce a re-used agent unless we need it and follow the new form of this commit.